fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Rewriter.java

```diff
@@ -306,24 +318,11 @@ public class Rewriter extends AbstractBatchJobExecutor {
                         // after eliminate outer join, we can move some filters to join.otherJoinConjuncts,
                         // this can help to translate plan to backend
                         topDown(new PushFilterInsideJoin()),
-                        topDown(new FindHashConditionForJoin()),
-                        topDown(new ExpressionNormalization())
```

Comment:
what's the consideration of this removing?

Reply:
It seems that deleting it is fine, because the PushFilterInsideJoin and FindHashConditionForJoin rules do not make any expression changes. All the necessary expression changes are done in PUSH_DOWN_FILTERS